go.mod

```diff
@@ -7,23 +7,23 @@ require (
 	github.com/google/go-containerregistry v0.0.0-20200413145205-82d30a103c0a // indirect
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/openzipkin/zipkin-go v0.2.2 // indirect
-	github.com/pkg/errors v0.8.1
+	github.com/pkg/errors v0.9.1
 	github.com/robfig/cron v1.2.0 // indirect
-	github.com/spf13/cobra v0.0.5
+	github.com/spf13/cobra v0.0.6
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.4.0
-	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
+	github.com/spf13/viper v1.6.2
+	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073
 	gomodules.xyz/jsonpatch/v2 v2.1.0 // indirect
 	gotest.tools v2.2.0+incompatible
 	k8s.io/api v0.17.4
 	k8s.io/apimachinery v0.17.4
-	k8s.io/cli-runtime v0.17.0
-	k8s.io/client-go v0.17.4
+	k8s.io/cli-runtime v0.17.3
+	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
```

Comment:
I'm not happy with this, as kn's code also use client-go and so the dependency should be controlled by the `kn`'s dependency not on behalf to the test infrastructure. Also, this should be also always a fixed version.